eng/versioning/version_client.txt

```diff
@@ -109,6 +109,7 @@ com.azure.cosmos.spark:azure-cosmos-spark_3-1_2-12;4.28.4;4.29.0-beta.1
 com.azure.cosmos.spark:azure-cosmos-spark_3-2_2-12;4.28.4;4.29.0-beta.1
 com.azure.cosmos.spark:azure-cosmos-spark_3-3_2-12;4.28.4;4.29.0-beta.1
 com.azure.cosmos.spark:azure-cosmos-spark_3-4_2-12;4.28.4;4.29.0-beta.1
+com.azure.cosmos.spark:azure-cosmos-spark_3-5_2-12;4.28.4;4.29.0-beta.1
```

Comment:
This is incorrect, we don't have any released version of spark 3.5 
so this should be -> 

```suggestion
com.azure.cosmos.spark:azure-cosmos-spark_3-5_2-12;4.29.0-beta.1;4.29.0-beta.1
```

Reply:
Would be good to check with engg team as well. @JimSuplizio - do you think the above suggestion makes sense here? We are adding a new spark package here.